core/controllers/suggestion_test.py

```diff
@@ -320,6 +322,49 @@ def test_suggestion_list_handler(self):
             )['suggestions']
         self.assertEqual(len(suggestions), 2)
 
+    def test_resubmit_rejected_suggestion(self):
+
+        self.login(self.EDITOR_EMAIL)
+        response = self.testapp.get('/explore/%s' % self.EXP_ID)
+        csrf_token = self.get_csrf_token_from_response(response)
+
+        suggestion = suggestion_services.query_suggestions(
+            [('author_id', self.author_id), ('target_id', self.EXP_ID)])[0]
+        suggestion_services.reject_suggestion(
+            suggestion, self.reviewer_id, 'reject message')
+        self.logout()
+
+        self.login(self.AUTHOR_EMAIL)
+        response = self.testapp.get('/explore/%s' % self.EXP_ID)
+        csrf_token = self.get_csrf_token_from_response(response)
+
+        self.put_json('%s/resubmit/%s' % (
+            feconf.SUGGESTION_ACTION_URL_PREFIX, suggestion.suggestion_id), {
+                'summary_message': 'summary message',
+                'action': u'resubmit',
+                'change': {
+                    'cmd': exp_domain.CMD_EDIT_STATE_PROPERTY,
+                    'property_name': exp_domain.STATE_PROPERTY_CONTENT,
+                    'state_name': 'State 1',
+                    'new_value': self.resubmit_change_content,
+                    'old_value': self.old_content
+                }
+            }, csrf_token=csrf_token)
+
+        suggestion = suggestion_services.query_suggestions(
+            [('author_id', self.author_id), ('target_id', self.EXP_ID)])[0]
+        self.assertEqual(
+            suggestion.status, suggestion_models.STATUS_IN_REVIEW)
+        self.assertEqual(
+            suggestion.change.new_value['html'], self.new_content['html'])
```

Comment:
Hmmm, shouldn't this be `self.resubmit_change_content['html']`. While testing the UI, I noticed that on resubmitting, the status changed to "in review" but the content stayed the same (before resubmit). I think this test is slightly off, and the backend needs to be fixed accordingly.